### ISSUE TITLE
can you state in README.md that one should restart sublime after installing easyclangcomplete?

### ISSUE THREAD

**Issue by T1n9** [niosus/EasyClangComplete#351]
I tried to install it today, tried modify include dirs, turn on/off different options, googled for a while. 
But nothing happened with my plugin, it seems it was just dead. 

Finally I decided to post a issue here, then I saw the "try restarting sublime". Tried, worked.

So can I ask you guys to add things like "please restart your sublime text" or "you may have to restart your sublime text" into README.md?

**Comment by niosus** [comment]
Oh, sorry for this experience. It should not be that way. Can I ask you to help me by telling me how to reproduce this? I was sure I have fixed the plugin so that it works out of the box right after the installation and it's my fault if it does not. Could you try to reinstall the plugin and copy the output in sublime's console here so that I know what is the issue?

**Comment by T1n9** [comment]
my setting is : Ubuntu 16.04.1, the new Sublime but I did not install it just extract the tar then run the executable.
I installed clang before the plugin, then after I installed the plugin, it simply does not show anything. I mean, everything is complete like before I installed it.

For debugging I did two tests:

1. I deleted the  sublime-text-3 folder(config folder) under $HOME/.config, then run sublime(the one I 
    installed before) 
    everything seems to be fresh new and unconfigured. 
    Installed the package control and easyclangcomplete. Test without restart -> it's working!

2. I downloaded a new tar of sublime text 3 and extracted it. Again, delete the config folder. Run the 
    sublime. Everything is fresh.
    Installed the package control and easyclangcomplete. Test without restart -> it failed.

So I suppose it has something to do with the executable it's self rather than the config files.
